config/owasp/suppressions.xml

```diff
@@ -77,7 +77,7 @@
     <notes>Wait for spring boot 3.10 upgrades</notes>
     <cve>CVE-2023-20883</cve>
   </suppress>
-  <suppress until ="2023-08-31">
+  <suppress until ="2023-09-07">
```

Comment:
Can put to the end of the next month if you want to e.g 30 Sept

Reply:
done